### PR TITLE
feat: clean up setting up azure-acquire

### PR DESCRIPTION
**Pull request by versey-sherry**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,6 +24,9 @@ def get_version(rel_path):
     version=get_version('azure_acquire/__init__.py'),
     packages=find_packages(),
     platforms=['mac', 'unix'],
+    install_requires=['click', 'Sphinx', 'coverage', 'flake8', 'python-dotenv>=0.5.1',
+                      'opencv-python', 'numpy', 'matplotlib', 'jupyter', 'pathlib2', 
+                      'tqdm', 'pyk4a', 'jupyter'],
```
Comment (wingillis) [inline setup.py]: most of these packages are unnecessary for running the acquisition code. Remove all packages that aren't directly called by `cli.py` and `util.py`

**Review by wingillis — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (versey-sherry) [reply setup.py]: 3558d2516ce3964e512f6e8779e3ea40590855f9

File: setup.py [hunk 2]
```diff
@@ -24,6 +24,7 @@ def get_version(rel_path):
     version=get_version('azure_acquire/__init__.py'),
     packages=find_packages(),
     platforms=['mac', 'unix'],
+    install_requires=['click', 'opencv-python', 'numpy', 'matplotlib', 'jupyter', 'pyk4a', 'jupyter'],
```
Comment (wingillis) [inline setup.py]: Jupyter is written twice.

Comment (wingillis) [reply setup.py]: Also, jupyter isn't used in the codebase - remove it

Comment (wingillis) [reply setup.py]: matplotlib isn't used in the codebase either - remove it

**Review by wingillis — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (versey-sherry) [reply setup.py]: i removed matplotlib but i want to keep jupyter in case users what to use jupyter notebook

**Pull request merged**